pkg/kn/commands/broker/create.go

```diff
@@ -36,6 +36,8 @@ var createExample = `
 // NewBrokerCreateCommand represents command to create new broker instance
 func NewBrokerCreateCommand(p *commands.KnParams) *cobra.Command {
 
+	var className string
```

Comment:
3 lines above there is an example added. could we add one with a `--class KafkaBroker` example, something like:

```
# Create a broker 'mybroker' in the 'myproject' namespace and with a broker class of 'KafkaBroker'
kn broker create mybroker --namespace myproject --broker KafkaBroker
````

(extending the last example, not adding a new one)